src/actions.js

```diff
@@ -167,22 +164,13 @@ export function fetchLocations(levels, type, parent) {
   return graphql(payload, `LOCATION_LOCATIONS_${type}`);
 }
 
-export function fetchLocationsStr(
-  mm,
-  level,
-  regions = null,
-  districts = null,
-  parent,
-  str,
-  first,
-) {
+export function fetchLocationsStr(mm, level, regions = null, districts = null, parent, str, first) {
   const types = mm.getConf("fe-location", "Location.types", ["R", "D", "W", "V"]);
```

Comment:
["R", "D", "W", "V"] -> I think this could refer to my first comment regarding 'const'.

Reply:
I tried to refactor that by it was causing some other issues. I would like to stay with this approach if you don't mind.

Reply:
ok, noted, therefore we can skip changing this, @olewandowski1